radix-engine-common/src/data/scrypto/model/address_resource_manager.rs

```diff
@@ -12,7 +12,8 @@ use utils::{copy_u8_array, ContextualDisplay};
 /// Represents a resource address.
 #[derive(Clone, Copy, PartialEq, Eq, Hash, PartialOrd, Ord)]
 pub enum ResourceAddress {
-    Normal([u8; ADDRESS_HASH_LENGTH]), // TODO: change to fungible & non-fungible
+    NonFungible([u8; ADDRESS_HASH_LENGTH]),
```

Comment:
Very niggly, but can we re-order them as Fungible then NonFungible for consistency?

Reply:
yup!